src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -77,7 +77,7 @@ class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-sv
 				></em>
 			<?php endif; ?>
 			<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
-				<?php echo wp_kses_post( $event->title ) ?>
+				<?php echo get_the_title( $event ); ?>
```

Comment:
I'm curious if we need to define if we're going to go with the template tags or echoing the getters on the templates. Maybe we could define this. Like, using `the_title` vs `echo get_the_title`

Reply:
Problem with `the_title` is that it doesnt accept the event ID, which is a problem for testing.